server/spec/fixtures/geojson.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Test utilities to generate GeoJSON data.
+ *
+ * @module server/spec/fixtures/geojson
+ */
+const chance = require('chance').Chance();
+
+/**
+ * Generates a GeoJSON point with random coordinates.
+ *
+ *     const geoJsonFixtures = require('../spec/fixtures/geojson');
+ *
+ *     geoJsonFixtures.point();  // { type: 'Point', coordinates: [ -76, 48 ] }
+ *
+ * @returns {object} A GeoJSON point.
+ */
+exports.point = function() {
+  return {
+    type: 'Point',
+    coordinates: exports.coordinates()
+  };
+};
+
+/**
+ * Generates a random pair of coordinates (longitude & latitude).
+ *
+ *     const geoJsonFixtures = require('../spec/fixtures/geojson');
+ *
+ *     geoJsonFixtures.coordinates();  // [ -76, 48 ]
+ *
+ * @returns {number[]} A GeoJSON coordinates pair.
+ */
+exports.coordinates = function() {
```

Comment:
Could this fixtures be used to generate locations data in the sample-data.js script ?
If yes, it might be useful to add some arguments to this function, like a way to pass a min/max for both latitude and longitude. Since BioPocket will be limited to the Onex territory, it would make sense to only generate points that are in the vicinity.

Reply:
(I didn't put this comment at the right place, hence the second request)